edge-modules/api-proxy-module/build.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
```

Comment:
Are we going to prepare/distribute images ourselves? If yes, check with @and-rewsmith. He did a great job making common build scripts for rust components.

Reply:
+1

Reply:
Is it wrong I actually re-used script from the module

Reply:
Yeah, we faced a bunch of issues with docker build image, so finally moved away from it

Reply:
Fixed, I use Andrew script now.